e2e/tests/stablex_test.rs

```diff
@@ -92,7 +97,10 @@ fn test_with_ganache() {
     let balance_after = tokens[1]
         .balance_of(accounts[0])
         .wait_and_expect("Cannot get balance after");
-    assert_eq!(balance_after - balance_before, 999_000.into())
+    assert_eq!(
+        balance_after - balance_before,
+        (999 * usd_price_in_fee).into()
+    )
```

Comment:
This assert fails with the standard solver because the balance change is 998.99... instead of exactly 999. Should we allow small fluctuations?

Reply:
Would it be cleaner to read out the used solver from the env variables and then make two different assertions, depending on the solver?

Reply:
Actually, a range check seems reasonable to me: 
Since we sometimes change minimal parameters in the solver, the expected number can change ( e.g. we might change the buffer for token shifting etc). Adjusting each time the unit tests seems like an unnecessary burden.

Reply:
A range check is fine with me although I don't think this test has to necessarily pass if you are using it wit a non-designated solver.